lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -558,6 +559,55 @@ class DecomposeAtenMatmulOp : public OpRewritePattern<AtenMatmulOp> {
 };
 } // namespace
 
+// ReLU6(x) = min(max(0, x), 6) = min(Relu(x), 6)
+static Value getRelu6Results(PatternRewriter &rewriter, Location loc,
+                             Value input) {
+  Type inputType = input.getType();
+  Value none = rewriter.create<Torch::ConstantNoneOp>(loc);
+  Value relu = rewriter.create<AtenReluOp>(loc, inputType, input);
+  Value emptyTensor = rewriter.create<AtenEmptyLikeOp>(
```

Comment:
It would be more efficient to create a rank 0 tensor to save the cost of memory allocation and write.

Reply:
I have created a rank 1 tensor of size 1.